app/src/main/java/com/swiezaki/swiezakimobile/network/RxErrorHandlingCallAdapterFactory.kt

```diff
@@ -0,0 +1,46 @@
+package com.swiezaki.swiezakimobile.network
```

Comment:
Are those two classes really need to be in one file?

Reply:
They don't need to. But they are clearly connected and this case it's very popular practice in Kotlin